modules/insticatorBidAdapter.js

```diff
@@ -190,31 +197,96 @@ function buildDevice(bidRequest) {
   return device;
 }
 
+function _getCoppa() {
+  return config.getConfig('coppa') === true ? 1 : 0;
```

Comment:
Can you get this similar to other regs with 'ortb2.regs.coppa' instead please.

Reply:
@ChrisHuie Thanks!

Updated the PR. Let me know if everything looks good.

Reply:
The test case failing is related to some other prebid core updates I guess, in file `/test/spec/e2e/outstream/basic_outstream_video_ad.spec.js`.

@ChrisHuie

Reply:
@ChrisHuie Thanks!

Updated the PR. Let me know if everything looks good.